app/cp_nigeria/helpers.py

```diff
@@ -80,3 +106,75 @@ def create_report_content(self, project):
         self.doc.add_paragraph("Here, the community will get information about who best to approach according to their"
                                "current situation (e.g. isolated or interconnected), their DisCo according to "
                                "geographical area etc.")
+
+
+def get_aggregated_cgs(project=None, community=None, SHS=True):
+    # list according to ConsumerType object ids in database
+    consumer_types = ["households", "enterprises", "public", "machinery"]
+    results_dict = {}
+
+    if SHS:
+        shs_consumers = get_shs_threshold(project)
```

Comment:
If here project is `None` a 404 error will be raised and in case of Community you do call this function with `project=None`
``` 
if options.community is not None:
     total_demand = get_aggregated_demand(community=options.community)
```